src/test/java/com/mongodb/hibernate/jdbc/MongoConnectionProviderTests.java

```diff
@@ -63,7 +63,7 @@ void testMongoClientCustomizerTakeEffect(boolean customizerAppliesConnectionStri
 
             // given
 
-            var hostInConnectionString = customizerAppliesConnectionString ? "my-host" : "localhost";
+            var hostInConnectionString = "host";
```

Comment:
As far as I can tell, there is no need to use two different values here.

Reply:
That is a leftover I forgot to remove when I refactored the testing logic to avoid internal class reliance.